k8s-apply/src/cmd/k8s_apply/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/signadot/plugins/k8s-apply/src/pkg/k8sapplier"
+	"github.com/signadot/plugins/k8s-apply/src/pkg/resources"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/discovery/cached/memory"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/restmapper"
+)
+
+func main() {
+	inputPath := flag.String("file", "", "input file")
+	flag.Parse()
+
+	// read the input file
+	if *inputPath == "" {
+		panic("input file not defined")
+	}
+	file, err := os.Open(*inputPath)
+	if err != nil {
+		panic(fmt.Sprintf("could not open input file, %v", err))
+	}
+
+	// creates an in-cluster config
+	cfg, err := rest.InClusterConfig()
+	if err != nil {
+		panic(err.Error())
+	}
+
+	// prepare a RESTMapper to find GVR
+	dc, err := discovery.NewDiscoveryClientForConfig(cfg)
+	if err != nil {
+		panic(err.Error())
+	}
+	mapper := restmapper.NewDeferredDiscoveryRESTMapper(memory.NewMemCacheClient(dc))
+
+	// prepare the dynamic client
+	dyn, err := dynamic.NewForConfig(cfg)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	ctx := context.Background()
```

Comment:
Maybe a timeout would be appropriate for this ctx, since it is doing queries.

Reply:
It's done here: https://github.com/signadot/plugins/pull/19/commits/78a29d8005f3c32f3af0f63308f2863e7e2dc592.